### PR TITLE
[install.sh] fix could not write config.xml file to the project/

### PR DESCRIPTION
**Pull request by kongtiaowang**
## Brief summary of changes
https://github.com/aces/Loris/issues/7096
fix centos group issue on project/

#### Testing instructions 
install Loris 
https://acesloris.readthedocs.io/en/latest/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.html
or https://acesloris.readthedocs.io/en/latest/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.html

### CODE REVIEW

**Comment by christinerogers** [comment]
Thanks Shen -- this looks good to me. 

@kongtiaowang Could you also issue the fix to the 23 branch?  Please also assign someone to test it since it would be good to know this is resolved on the 23 branch before (e.g.) HBM this month. 

@VietDuyHoang Could you take a look at this, and your install notes, and let us know here if you think this will resolve the issue you experienced, or was there another issue at root?   Thanks.

File: tools/install.sh [hunk 1]
```diff
@@ -138,6 +138,7 @@ elif [[ " ${redhat[*]} " == *" $os_distro "* ]]; then
     sudo chown apache.apache ../smarty/templates_c
     # Make Apache the group for project directory, so that the web based install
     # can write the config.xml file.
+    sudo chgrp www-data ../project
```
Comment (driusan) [inline tools/install.sh:141]: This is different than the group used for `templates_c` 2 lines up.

**Comment by VietDuyHoang** [comment]
> @VietDuyHoang Could you take a look at this, and your install notes, and let us know if there was another issue at root? Thanks.

@kongtiaowang  I'm not sure if adding the `chgrp` command will address the issue, because when I attempted to install LORIS, the group ownership for `/project` was already  `www-data`:

`drwxrwx---   8 lorisadmin www-data     4096 May 25 14:42 project`

**Comment by kongtiaowang** [comment]
This fix only works on Centos. 
VietDuyHoang did not follow the standard installation process completely, which caused permission problems.

**Comment by christinerogers** [comment]
Thanks Shen. Looks like it's ready for review then. 
If you think it's relevant, could you also commit this fix to the 23 branch? thanks

**Pull request merged**